SCons/Tool/suncxx.py

```diff
@@ -87,12 +85,13 @@ def get_package_info(package_name, pkginfo, pkgchk):
                               'stderr': DEVNULL}
                 popen_args['universal_newlines'] = True
                 p = subprocess.Popen([pkgchk, '-l', package_name],
-                                     **popen_args)
+                                     universal_newlines=True,
+                                     stdout=subprocess.PIPE,
+                                     stderr=DEVNULL)
             except EnvironmentError:
```

Comment:
Python 3 has eliminated `EnvironmentError` as a separate category, suggest using `OSError` or the much more specific `FileNotFoundError` - though that's really a cleanup task not required for this PR....

Reply:
use of `EnvironmentError` seems to have wider scope 
```
robb@mars:/raid/checkouts-raid/robb/scons/scons-2 (unix_py3_str)$ git grep ' EnvironmentError'
CHANGES.txt:    - Fix issue #3415 - Update remaining usages of EnvironmentError to SConsEnvironmentError
CHANGES.txt:    - Fix Issue #3350 - SCons Exception EnvironmentError is conflicting with Python's EnvironmentError.
CHANGES.txt:    - Fix Issue #3349 - SCons Exception EnvironmentError is conflicting with Python's EnvironmentError.
SCons/Action.py:    except EnvironmentError as e:
SCons/ActionTests.py:        except EnvironmentError:
SCons/CacheDir.py:        except EnvironmentError:
SCons/CacheDir.py:    except EnvironmentError:
SCons/Node/FS.py:        except EnvironmentError as e:
SCons/Node/FS.py:        except (KeyError, EnvironmentError):
SCons/Node/__init__.py:    except EnvironmentError as e:
SCons/Platform/win32.py:    except (OSError, EnvironmentError) as e:
SCons/Scanner/C.py:        except EnvironmentError as e:
SCons/Scanner/C.py:        except EnvironmentError:
SCons/Script/Interactive.py:        except EnvironmentError as e:
SCons/Taskmaster.py:            except EnvironmentError as e:
SCons/Tool/intelc.py:        except EnvironmentError:
SCons/Tool/suncxx.py:        except EnvironmentError:
SCons/Tool/suncxx.py:        except EnvironmentError:
SCons/Tool/suncxx.py:            except EnvironmentError:
runtest.py:        except EnvironmentError:
testing/framework/TestCmdTests.py:        except EnvironmentError:
testing/framework/TestCmdTests.py:        except EnvironmentError:
testing/framework/TestCmdTests.py:        except EnvironmentError:
testing/framework/TestSCons.py:        except EnvironmentError:
```
and I agree this should probably be addressed, and probably not in this PR.

Reply:
@mwichmann although if we would prefer to use `FileNotFoundError` here, that might go in this PR - what do you think?

Reply:
I'm actually not yet clear on whether there can be _other_ failure modes from Popen than `FileNotFoundError`, which presumably means the command you asked to run can't be found via the existing PATH...  so hard to answer.

Reply:
Ok, it's probably better to leave it and preserve current behavior until there is a more urgent reason to change it.